modules/social_features/social_topic/social_topic.install

```diff
@@ -203,6 +203,20 @@ function social_topic_update_120002(): void {
   }
 }
 
+/**
+ * Ensure the sidebar block "Newest topics" will be displayed on all groups.
+ */
+function social_topic_update_120003(): string {
```

Comment:
Since we plan to release it in version 12.2.x it's better to rename the hook update number to the `_122001`